diff --git a/conf/httpd.conf b/conf/httpd.conf
index 602141d..4cfd2c0 100644
--- a/conf/httpd.conf
+++ b/conf/httpd.conf
@@ -156,7 +156,7 @@ LoadModule proxy_balancer_module modules/mod_proxy_balancer.so
 #LoadModule session_cookie_module modules/mod_session_cookie.so
 #LoadModule session_crypto_module modules/mod_session_crypto.so
 #LoadModule session_dbd_module modules/mod_session_dbd.so
-#LoadModule slotmem_shm_module modules/mod_slotmem_shm.so
+LoadModule slotmem_shm_module modules/mod_slotmem_shm.so
 #LoadModule slotmem_plain_module modules/mod_slotmem_plain.so
 #LoadModule ssl_module modules/mod_ssl.so
 #LoadModule optional_hook_export_module modules/mod_optional_hook_export.so
