diff --git a/conf/httpd.conf b/conf/httpd.conf
index 4cfd2c0..b43c045 100644
--- a/conf/httpd.conf
+++ b/conf/httpd.conf
@@ -563,11 +563,11 @@ SSLRandomSeed connect builtin
   AllowEncodedSlashes NoDecode
 
   ProxyRequests Off
-  <Proxy *>
-    Order deny,allow
-    Allow from all
-  </Proxy>
   ProxyPreserveHost On
-  ProxyPass /xwiki http://tomcat:8080/xwiki nocanon
-  ProxyPassReverse /xwiki http://tomcat:8080/xwiki
+
+  <Proxy balancer://mycluster>
+  BalancerMember ajp://tomcat:8009 route=jvm1
+  ProxySet stickysession=JSESSIONID
+  </Proxy>
+  ProxyPass /xwiki balancer://mycluster/xwiki
 </VirtualHost>
