diff --git a/conf/httpd.conf b/conf/httpd.conf
index b43c045..003ba5c 100644
--- a/conf/httpd.conf
+++ b/conf/httpd.conf
@@ -167,7 +167,7 @@ LoadModule slotmem_shm_module modules/mod_slotmem_shm.so
 #LoadModule http2_module modules/mod_http2.so
 #LoadModule proxy_http2_module modules/mod_proxy_http2.so
 #LoadModule md_module modules/mod_md.so
-#LoadModule lbmethod_byrequests_module modules/mod_lbmethod_byrequests.so
+LoadModule lbmethod_byrequests_module modules/mod_lbmethod_byrequests.so
 #LoadModule lbmethod_bytraffic_module modules/mod_lbmethod_bytraffic.so
 #LoadModule lbmethod_bybusyness_module modules/mod_lbmethod_bybusyness.so
 #LoadModule lbmethod_heartbeat_module modules/mod_lbmethod_heartbeat.so
